transcom/mymove
# Int b 19925 sorting branch param to uppercase

This is a redo of 19925 which Michael had partially done. The change I added was making it work for the TIO as well - was working for the TOO as of the first integration PR from this branch.

Now, should work for TOO and TIO.

Check out the previous int PR:
## [Previous INT PR](https://github.com/transcom/mymove/pull/12924)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A948237)

To test:
- View the TIO queue
- Apply the Army branch filter
- Make a note of the number of moves
- Grab a move locator that appears with the filter on
- In the database, go to that move -> order -> service member
- Change affiliation from ARMY to Army (or even ArMy)
- Save
- Go back to the TIO queue
- Apply the Army branch filter again 
- Make sure the move you modified for the service member appears AND the number of moves is the same as the first time you applied the branch filter

Do the same for the TOO
